app/services/adapters/structs/crime_application.rb

```diff
@@ -4,6 +4,12 @@ module Adapters
   module Structs
     class CrimeApplication < LaaCrimeSchemas::Structs::CrimeApplication
       include TypeOfApplication
+      # `is_means_tested` is not part of Schema, requires calculation
+      # rubocop:disable Naming/PredicateName
+      def is_means_tested
```

Comment:
what aboud just `means_tested?` I think it means the same as `is_means_tested?` and we could lose the rubocop opt out